kobuki_node/include/kobuki_node/kobuki_ros.hpp

```diff
@@ -159,61 +182,85 @@ class KobukiRos
   void publishRawInertia();
   void publishSensorState();
   void publishDockIRData();
-  void publishVersionInfo(const VersionInfo &version_info);
+  void publishVersionInfo(const kobuki::VersionInfo &version_info);
   void publishControllerInfo();
-  void publishButtonEvent(const ButtonEvent &event);
-  void publishBumperEvent(const BumperEvent &event);
-  void publishCliffEvent(const CliffEvent &event);
-  void publishWheelEvent(const WheelEvent &event);
-  void publishPowerEvent(const PowerEvent &event);
-  void publishInputEvent(const InputEvent &event);
-  void publishRobotEvent(const RobotEvent &event);
+  void publishButtonEvent(const kobuki::ButtonEvent &event);
+  void publishBumperEvent(const kobuki::BumperEvent &event);
+  void publishCliffEvent(const kobuki::CliffEvent &event);
+  void publishWheelEvent(const kobuki::WheelEvent &event);
+  void publishPowerEvent(const kobuki::PowerEvent &event);
+  void publishInputEvent(const kobuki::InputEvent &event);
+  void publishRobotEvent(const kobuki::RobotEvent &event);
 
 
   // debugging
-  void rosDebug(const std::string &msg) { ROS_DEBUG_STREAM("Kobuki : " << msg); }
-  void rosInfo(const std::string &msg) { ROS_INFO_STREAM("Kobuki : " << msg); }
-  void rosWarn(const std::string &msg) { ROS_WARN_STREAM("Kobuki : " << msg); }
-  void rosError(const std::string &msg) { ROS_ERROR_STREAM("Kobuki : " << msg); }
+  void rosDebug(const std::string &msg) { RCLCPP_DEBUG(get_logger(), "Kobuki : %s", msg.c_str()); }
+  void rosInfo(const std::string &msg) { RCLCPP_INFO(get_logger(), "Kobuki : %s", msg.c_str()); }
+  void rosWarn(const std::string &msg) { RCLCPP_WARN(get_logger(), "Kobuki : %s", msg); }
+  void rosError(const std::string &msg) { RCLCPP_ERROR(get_logger(), "Kobuki : %s", msg); }
   void rosNamed(const std::vector<std::string> &msgs) {
-    if (msgs.size()==0) return;
-    if (msgs.size()==1) { ROS_INFO_STREAM("Kobuki : " << msgs[0]); }
-    if (msgs.size()==2) {
-      if      (msgs[0] == "debug") { ROS_DEBUG_STREAM("Kobuki : " << msgs[1]); }
-      else if (msgs[0] == "info" ) { ROS_INFO_STREAM ("Kobuki : " << msgs[1]); }
-      else if (msgs[0] == "warn" ) { ROS_WARN_STREAM ("Kobuki : " << msgs[1]); }
-      else if (msgs[0] == "error") { ROS_ERROR_STREAM("Kobuki : " << msgs[1]); }
-      else if (msgs[0] == "fatal") { ROS_FATAL_STREAM("Kobuki : " << msgs[1]); }
+    if (msgs.size() == 0) {
+      return;
+    }
+    if (msgs.size() == 1) {
+      RCLCPP_INFO(get_logger(), "Kobuki : %s", msgs[0]);
+    }
+    if (msgs.size() == 2) {
+      if (msgs[0] == "debug") {
+        rosDebug(msgs[1]);
+      }
+      else if (msgs[0] == "info") {
+        rosInfo(msgs[1]);
+      }
+      else if (msgs[0] == "warn") {
+        rosWarn(msgs[1]);
+      }
+      else if (msgs[0] == "error") {
+        rosError(msgs[1]);
+      }
+      else if (msgs[0] == "fatal") {
+        RCLCPP_FATAL(get_logger(), "Kobuki : %s", msgs[1]);
```

Comment:
Probably should be consistent, one way or another. Those `rosDebug/...` methods above are expressly so they can be attached to sigslot callbacks, but if we're going to use them here instead of the macros, let's use them consistently.

Reply:
Sounds good.  I added `rosFatal()` in 9605787.